style.css

```diff
@@ -0,0 +1,275 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins');
+
+html,
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Poppins;
+  font-size: 10px;
+  font-weight: 300;
+  line-height: 3.5em;
+}
+
+h1 {
+  font-size: 9em;
+  text-align: center;
+}
+
+h2 {
+  color: white;
+  text-shadow: #020819 8px -20px 9px;
+  font-size: 5em;
+  text-align: center;
+  text-transform: uppercase;
+  line-height: 4em;
+}
+
+h3 {
+  font-size: 4.2em;
+  text-align: center;
+  line-height: 1em;
+}
+
+h4 {
+  font-size: 1.5em;
+  letter-spacing: 0.4px;
+  line-height: 1em;
+  margin-bottom: 15px;
+}
+
+h5 {
+  font-size: 1.2em;
+  line-height: 1em;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+header > div {
+  padding: 0 25px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: 1px solid lightgray;
+}
+
+nav {
+  width: 600px;
+}
+
+nav ul {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+nav a {
+  text-decoration: none;
+  color: black;
+}
+
+.blackHeart {
+  width: 20px;
+  margin-right: 1rem;
+}
+
+hr {
+  width: 100%;
+  border-top: 2px solid #f6f4f2;
+}
+
+#search {
+  display: flex;
+  justify-content: center;
+}
+
+.npm-logo {
+  width: 50px;
+  margin: 0 10px 0 0;
+}
+
+.magnifying-glass {
+  width: 20px;
+  background-color: #f3f2f3;
+  padding: 10px 10px;
+  margin: 2px 0 0 0;
+}
+
+.input-text {
+  width: 900px;
+  border: 3px solid #f3f2f3;
+  color: rba(0,0,0,.05);
+  background-color: #f3f2f3;
+  padding: 10px 10px;
+  margin: 3px 0 3px 0;
+}
+
+.search-btn {
+  font-size: 1.2em;
+  background-color: #fb3e44;
+  color: white;
+  border: 0px solid #fbced4;
```

Comment:
The search button is a bit more narrow than the rest of the input field - the difference is the width of the border. If you change the color of the border to the as as background-color, it will even out.